src/handlers/text.ts

```diff
@@ -22,10 +22,10 @@ export const diameterTextHandler = (context: tfTypes.ContextMessageUpdate) => {
 }
 
 export const lentaSaleTextHandler = async (context: tfTypes.ContextMessageUpdate) => {
-  const matches = context.message.text.toLowerCase().match(/lenta/)
+  const matches = context.message.text.toLowerCase().match(/lentasale/)
   if (matches) {
     const saleList =  await lenta()
-    context.reply(`Found ${saleList.length} items`)
+    // context.reply(`Found ${saleList.length} items`)
```

Comment:
Why did you commented this line?

Reply:
Becasue I don't care how much seles were found. And delay between this message and actual result is also annoying. IMHO.